finality-provider/config/poller.go

```diff
@@ -4,7 +4,7 @@ import "time"
 
 var (
 	defaultBufferSize        = uint32(1000)
-	defaultPollingInterval   = 20 * time.Second
+	defaultPollingInterval   = 100 * time.Millisecond
```

Comment:
i think the polling interval shouldn't be less than the chain block-producing time. OP chain is usually 2-3 seconds. what's the block time for Babylon chain?

Reply:
I would argue that we should have a short polling interval for a quick catching up. Would `ms` level interval a concern in practice?

Reply:
Catching up is not a normal case. If it's way behind, people should just change the config file temporarily to speed up. imo default value should be for normal  submission cases

Reply:
Makes sense. Changed it to `1s` and added notes there